test/language/expressions/import.meta/syntax/invalid-assignment-target-update-expr.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2018 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-static-semantics-static-semantics-assignmenttargettype
+description: >
+  import.meta is not a valid assignment target.
+info: |
+  Static Semantics: AssignmentTargetType
+
+    ImportMeta:
+      import.meta
+
+    Return invalid.
```

Comment:
I'm glad you're already using up to date spec text, while the spec proposal still has the [old format](https://tc39.github.io/proposal-import-meta/#sec-static-semantics-static-semantics-isvalidsimpleassignmenttarget).